embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/injection/ModuleInitBootstrapper.kt

```diff
@@ -350,10 +352,17 @@ internal class ModuleInitBootstrapper(
                     }
 
                     postInit(NativeFeatureModule::class) {
+                        val configService = configModule.configService
                         serviceRegistry.registerServices(
                             lazy { nativeFeatureModule.nativeThreadSamplerService }
                         )
-                        nativeFeatureModule.nativeCrashHandlerInstaller?.install()
+
+                        if (configService.autoDataCaptureBehavior.isNativeCrashCaptureEnabled()) {
```

Comment:
Do we need to check this here? We're already checking the ndk config in the feature module. If it's not enabled, it'll return null.

Reply:
Removed this redundant check